sources/web/datalab/static/dark.css

```diff
@@ -106,6 +106,15 @@ input {
   color: #333;
 }
 
+/* Terminal Page */
```

Comment:
Are there corresponding changes in light.css to keep the terminal color theme consistent with the rest of datalab?